src/ibattle.cpp

```diff
@@ -754,6 +754,14 @@ void IBattle::SetHostGame(const std::string& gamename, const std::string& hash)
 	m_game_loaded = gamename.empty();
 	m_host_game.name = gamename;
 	m_host_game.hash = hash;
+	
+	gameNameWithoutVersion = gamename.substr(0,gamename.find_last_of(" "));
```

Comment:
this relies on games having no space in its version. this looks like a predetermined breaking point.

Reply:
In those cases it will assume as prefix a larger string that contains part of the version.

at the moment that's only used to figure out the colors, so this won't really have any impact other than some weirdness when sorting and coloring the game (it may assume different versions as different games).

I don't remember any such case recently on the lobby server. Can you find any game on springfiles with spaces on the version?

What do you suggest?